src/main/java/by/exadel/internship/controller/FormController.java

```diff
@@ -21,20 +22,20 @@
 @Api(tags = "Endpoints for Form")
 public class FormController {
 
-    private final FormService formService;
+    private final FormService FORM_SERVICE;
```

Comment:
@Maksim-maevsky Could you share with me the code style convenshion, where i can see this rule?

Reply:
I changed to lower case